lib/puppet/provider/cpdb_update/cpdb_update.rb

```diff
@@ -28,22 +23,16 @@ def migrate_database
     )
   end
 
-  def version_file
-    "/var/lib/candlepin/.puppet-candlepin-rpm-version"
```

Comment:
Should we ensure this is absent to leave less stuff behind?

Reply:
Good idea- will do.